cms/static/js/collections/group.js

```diff
@@ -5,17 +5,20 @@ function (_, str, Backbone, gettext, GroupModel) {
     'use strict';
     var GroupCollection = Backbone.Collection.extend({
         model: GroupModel,
-        comparator: 'order',
+
         /*
-         * Return next index for the model.
-         * @return {Number}
+         * Sort groups by name, alphabetically.
          */
-        nextOrder: function() {
-            if(!this.length) {
+        comparator: function(first, second) {
```

Comment:
I think this would be simpler if you use the one argument version of a comparator (a "sortBy" comparator):

http://documentcloud.github.io/backbone/#Collection-comparator

This would then be something like (untested);

    comparator: function(group) {
        return group.get('name').toLowerCase();
    }